Testing/ContinuousIntegration/AzurePipelinesBatch.yml

```diff
@@ -58,7 +58,7 @@ jobs:
 
           curl -L https://github.com/InsightSoftwareConsortium/ITK/releases/download/v$(ExternalDataVersion)/InsightData-$(ExternalDataVersion).tar.gz -O
           cmake -E tar xfz InsightData-$(ExternalDataVersion).tar.gz
-          cmake -E rename InsightToolkit-$(ExternalDataVersion)/.ExternalData/MD5 $(Build.SourcesDirectory)/.ExternalData/MD5
+          cmake -E rename InsightToolkit-$(ExternalDataVersion)/.ExternalData/CID $(Build.SourcesDirectory)/.ExternalData/CID
```

Comment:
For my understanding, can you please explain why "MD5" is replaced with "CID"?

Reply:
See https://github.com/InsightSoftwareConsortium/ITK/pull/4215 and the related issue.